examples/broadcasting/tests/broadcasting.yaml

```diff
@@ -1,35 +1,18 @@
----
 program: broadcasting
 inputs:
-  B_1:
-    SecretInteger: "3"
-  A_0:
-    SecretInteger: "3"
-  C_1:
-    SecretInteger: "3"
-  B_0:
-    SecretInteger: "3"
-  D_0:
-    SecretInteger: "3"
-  C_2:
-    SecretInteger: "3"
-  C_0:
-    SecretInteger: "3"
-  A_2:
-    SecretInteger: "3"
-  A_1:
-    SecretInteger: "3"
-  D_1:
-    SecretInteger: "3"
-  B_2:
-    SecretInteger: "3"
-  D_2:
-    SecretInteger: "3"
-
+  B_1: 3
```

Comment:
interesting change. how do we know if these values are public/secret and signed/unsigned?
can we no longer customize this is these YAML files?

Reply:
Basically, the value is introduced into the specific type expected by the Nada program. So it takes it already from the Nada Program.